app/backend/src/couchers/servicers/communities.py

```diff
@@ -260,18 +261,14 @@ def ListEvents(self, request, context):
                 .filter(Event.owner_cluster == node.official_cluster)
             )
 
-            if not request.past:
```

Comment:
The communities `ListEvent` request object doesn't seem to have this `past` field so I removed it, but this still doesn't seem to be working...

Reply:
In my local db I got an online event happening next week, but this is giving me an empty list on the frontend...

Reply:
I'm guessing that ListEventReq in communities.proto was supposed to have a past boolean. LMK if you want me to add it for you.

Reply:
Yea probably... I added it to the proto and put the deleted code back